app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.errors import (VaccineError, NotWearingMaskError)
+
```

Comment:
There is no need to add a blank line between imports from the same imports group according to PEP-8 (imports relate to the same group of Local application/library specific imports).